scripts/test_update_from_tag.sh

```diff
@@ -90,13 +92,15 @@ docker_pgcmd() {
 }
 
 docker_pgscript() {
-    docker_exec $1 "psql -h localhost -U postgres -v ON_ERROR_STOP=1 -f $2"
+    local database=${3:-postgres}
+    docker_exec $1 "psql -h localhost -U postgres -d $database -v ON_ERROR_STOP=1 -f $2"
 }
 
 docker_pgtest() {
+    local database=${3:-single}
```

Comment:
With current implementation the argument will be always set, so may be just use `$3` directly?